### PR TITLE
Simplify and improve text readability

### PR DESCRIPTION
**Pull request by albertmir**
Many improvements

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,7 +65,7 @@ Always be considerate and respectful, specifically:
 
 Instead:
 
-- ask questions, assuming that everyone is intelligent and well-meaning
```
Comment (albertmir) [inline README.md:68]: Can `intelligent` be an exclusive word?

File: README.md [hunk 2]
```diff
@@ -228,7 +233,7 @@ Always consider the potential impact of your words, not just their original inte
 
 Instead of using words like: "your", "mine", "not mine", which could be interpreted as finger-pointing, prefer the more inclusive "our" and "we" ones.
 
-Rephrase any comment to be more collaborative, showing that you share ownership of the code being modified.
+Rephrase any comment to be more collaborative. After all, it's our code. Do you remember?
```
Comment (albertmir) [inline README.md:236]: This expression is yours 😉

File: README.md (hunk 1 above)
Comment (mawrkus) [reply README.md:68]: How can it be if you assume that _everyone_ is? :)

File: README.md [hunk 3]
```diff
@@ -328,7 +333,7 @@ In the following interactions, can you spot which guideline applies?
 > It’s a polyfill used in lazyLoad.js
 >
 > Jamie:
-> Yes I got that, but why not importing there instead of here? I know that the import is not executed always in lazyload.js, but I see it weird.. Maybe a require() instead?
```
Comment (mawrkus) [inline README.md:331]: Though your suggestions are correct, I'd prefer to leave these examples exactly as I found them in real code reviews. The idea of this section is to find which guidelines would apply to them. It looks you've found quite some!

**Review by mawrkus — CHANGES_REQUESTED** [review]
Again, thanks for your attentive review Albert! I'd like to discuss with you about some of your comments as I think some of your suggestions change slightly the meaning/intent of the text. I'll find you outside of GitHub in a few weeks, ok?

File: README.md [hunk 4]
```diff
@@ -184,7 +187,7 @@ Always keep in mind that it's not about you, it's about a better code, a better
 
 If a reviewer says that they don't understand something in your code, your first response should be to clarify the code itself. If the code can't be clarified, add a code comment that explains why the code is there. If a comment seems pointless, only then should your response be an explanation in the code review.
 
-If a reviewer didn't understand some piece of your code, it's likely other future readers of the code won't understand either. Writing a response in the code review doesn't help future code readers, but clarifying your code or adding code comments does help them.
+If a reviewer don't understand some piece of your code, other future readers likely won't understand it either. Writing a response in the code review doesn't help future code readers, but clarifying your code or adding code comments does help them.
```
Comment (mawrkus) [inline README.md:190]: "doesn't" ?

**Review by mawrkus — APPROVED** [review]

**Comment by mawrkus** [comment]
Thanks again!

**Pull request merged**